css/style.css

```
@@ -0,0 +1,181 @@
+.bemvindo {
+    height: 500px;
+    width: 1250px;
+}
+
+body {
+    background: #000;
+}
+
+button {
+    background: #FFA500;
+    border: 1px solid #000;
+    color: #000;
+    padding: 5px 20px 5px 20px;
+}
+
+button:hover {
+    background: #000;
+    color: #fff;
+}
+
+.container {
+	display: flex;
+    margin: 0 auto;
+}
+
+#contato {
+    background: #FFA500;
+    padding: 50px;
+    text-align: center;
+}
+
+.fab {
+    color: #000;
+    background: yellow;
+    border-radius: 50%;
+    padding: 10px;
+    margin: 5px;
+}
+
+figcaption {
+    color: #ffa500;
+}
+
+.fundo-slide {
+    width: 100%;
+    height: 400px;
+}
+
+h2 {
+    color: #e6197f;
+    text-align: center;
+}
+
+h3 {
+    color: #FFA500;
+    text-align: center;
+}
+
+h4 {
+    color: #000;
+}
+
+h5 {
+	color: #ffa500;
+}
+
+.img-portifolio {
+	width: 250px;
+}
+
+.img-servicos {
+    border: 2px solid;
+    border-radius: 50%;
+    color: #fff;
+    width: 150px;
+}
+
+.img-slide {
```

```suggestion
.img-slide {
     width: 280px;
    height: 250px;
}
``` 
Usando o `flexbox` para alinhar os elementos verticalmente você não precisa mais usar o `position absolute`. Basta deixar apenas a largura e altura da sua imagem.